project_one.html

```diff
@@ -15,10 +15,84 @@
   </head>
   <body>
     <section class="container">
+      <div class="pageone">
+      <header class="header">
+        <nav class="logo">
+          <a href="#" class="logo1"> My Logo </a>
+          <ul>
+            <li> <a href="#">Portfolio</a> </li>
+            <li> <a href="#">About</a> </li>
+            <li> <a href="#">Contact</a> </li>
+          </ul>
```

Comment:
- [ ] At the moment, your navigation links are not aligned properly to the right as that of the original design. Please ensure to stick to the design guidelines

### Your Design
![image](https://user-images.githubusercontent.com/43995830/160106858-9b8d8321-782a-43c5-8a92-87e3ed773e77.png)

### Original Design
![image](https://user-images.githubusercontent.com/43995830/160106952-0486648b-3265-4733-8b2a-361dd5abf78d.png)

Reply:
Thank you sir for highlighting my mistake.